### PR TITLE
Add 1.11.3 release notes

### PR DESCRIPTION
**Pull request by ryantking**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [x] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.11.x/announcing-1.11.3/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Announcing Istio 1.11.3
+linktitle: 1.11.3
+subtitle: Patch Release
+description: Istio 1.11.3 patch release.
+publishdate: 2020-07-29
```
Comment (ericvn) [inline content/en/news/releases/1.11.x/announcing-1.11.3/index.md:6]: Date is wrong

File: content/en/news/releases/1.11.x/announcing-1.11.3/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+- **Fixed** `istioctl x workload` command output to set the correct `discoveryAddress` for revisioned control-planes.
+  ([Issue #34058](https://github.com/istio/istio/issues/34058))
+
+- **Fixed** `BenchmarkAnalyzersArtificialBlankData100` test. Also, fixed the gateway analyzer message reporting if there is no selector in the gateway spec.
+  ([Issue #35093](https://github.com/istio/istio/issues/35093))
+
+- **Fixed** an issue causing memory to not be freed after XDS clients disconnect.
+
+- **Fixed** an issues occurring when multiple `VirtualServices` with the same name exist in different namespaces.
```
Comment (ericvn) [inline content/en/news/releases/1.11.x/announcing-1.11.3/index.md:39]: an issue

File: content/en/news/releases/1.11.x/announcing-1.11.3/index.md (hunk 1 above)
Comment (ryantking) [reply content/en/news/releases/1.11.x/announcing-1.11.3/index.md:6]: Tomorrow?

Comment (ericvn) [reply content/en/news/releases/1.11.x/announcing-1.11.3/index.md:6]: well July in 2020 is wrong. Not sure if plan is today or tomorrow.

File: content/en/news/releases/1.11.x/announcing-1.11.3/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+- **Updated** to allow specifying NICs that bypassing traffic capture in Istio iptables.
+  ([Issue #34753](https://github.com/istio/istio/issues/34753))
+
+- **Added** values to the Istio Gateway Helm charts for configuring annotations on the `ServiceAccount`.  Can be used to enable [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) on AWS EKS.
+
+- **Fixed** `istioctl analyze` command to not output [IST0132] message when analyzing the gateway of the virtual service.
+  ([Issue #34653](https://github.com/istio/istio/issues/34653))
+
+- **Fixed** a bug using Service's pointer address to get its instances in case that a sidecar's egress listener has port.
```
Comment (ericvn) [inline content/en/news/releases/1.11.x/announcing-1.11.3/index.md:26]: This doesn't sound quite right and I'm not sure what the user would have actually seen that is being fixed.

Maybe: Fixed a bug using a Service's pointer address to get its instances in the case where a sidecar's egress listener has a port.

File: content/en/news/releases/1.11.x/announcing-1.11.3/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+- **Fixed** a bug in "image: auto" analyzer causing it to fail to take into account Deployment namespace.
```
Comment (ericvn) [inline content/en/news/releases/1.11.x/announcing-1.11.3/index.md:28]: Fixed a bug in **the** ... **the** Deployment namespace.

File: content/en/news/releases/1.11.x/announcing-1.11.3/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: Announcing Istio 1.11.3
+linktitle: 1.11.3
+subtitle: Patch Release
+description: Istio 1.11.3 patch release.
+publishdate: 2021-09-24
```
Comment (jwendell) [inline content/en/news/releases/1.11.x/announcing-1.11.3/index.md:6]: this should be today

File: content/en/news/releases/1.11.x/announcing-1.11.3/index.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+- **Fixed** a bug using a Service's pointer address to get its instances in the case where a sidecar's egress listener has a port.
+
+- **Fixed** a bug in the "image: auto" analyzer causing it to fail to take into account the Deployment namespace.
+  ([Issue #34929](https://github.com/istio/istio/issues/34929))
+
+- **Fixed** `istioctl x workload` command output to set the correct `discoveryAddress` for revisioned control-planes.
+  ([Issue #34058](https://github.com/istio/istio/issues/34058))
+
+- **Fixed** `BenchmarkAnalyzersArtificialBlankData100` test. Also, fixed the gateway analyzer message reporting if there is no selector in the gateway spec.
```
Comment (jwendell) [inline content/en/news/releases/1.11.x/announcing-1.11.3/index.md:34]: Remove the "`BenchmarkAnalyzersArtificialBlankData100 test. Also, fixed the`" part. We don't need to document the fix in a test.

**Review by jwendell — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Pull request merged**